src/plugins/mf_priority.cpp

```diff
@@ -204,6 +216,74 @@ static int validate_cb (flux_plugin_t *p,
                                     "user does not belong to specified bank");
     }
 
+    mj_it = users_mj.find (userid);
+
+    if (bank == NULL) {
+        def_bank_it = users_def_bank.find (userid);
+        mj_inner_it = mj_it->second.find (def_bank_it->second);
```

Comment:
Should you check for failure from `std::map::find` here? If these functions are guaranteed to succeed due to checks in `job.validate` then a comment to that effect might help. (Though I still wonder if the error condition should be checked here to avoid a broker crash. Someone with more C++ experience might be able to tell you)